Sofa/Component/Collision/Detection/Algorithm/src/sofa/component/collision/detection/algorithm/IncrSAP.h

```diff
@@ -207,12 +207,19 @@ class SOFA_COMPONENT_COLLISION_DETECTION_ALGORITHM_API IncrSAP :
      * Used in initialisation of IncrSAP. It clears all the IncrSAP fields.
      */
     void purge();
-
-
+    SOFA_ATTRIBUTE_DEPRECATED__RENAME_DATA()
     Data<bool> bDraw; ///< enable/disable display of results
 
+
+    SOFA_ATTRIBUTE_DEPRECATED__RENAME_DATA()
     Data< type::fixed_array<type::Vec3,2> > box; ///< if not empty, objects that do not intersect this bounding-box will be ignored
 
+
+
+    Data<bool> d_bDraw; ///< enable/disable display of results
```

Comment:
Actually, `bDraw` is never used. So I would be in favor to deprecate it without renaming. You must use `DeprecatedData` (see https://www.sofa-framework.org/community/doc/contributing-to-sofa/lifecycle-mechanisms/). The method `setDraw` is also to be deprecated.